analyzers/tests/SonarAnalyzer.UnitTest/Common/Paths.cs

```diff
@@ -18,22 +18,13 @@
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
 
-using System.IO;
-
 namespace SonarAnalyzer.UnitTest.Common
 {
-    public static class PathResolver
+    public static class Paths
     {
-        public static readonly string RelativePathToTestProjectRoot = @"..\..\..\";
+        public static string RelativePathToTestProjectRoot { get; private set; } = @"..\..\..\";
```

Comment:
When seeing all the usages below, I don't see a good reason for this to be a relative path anymore. Let's init it with `Path.GetFullPath` and simplify all the ussages.
```suggestion
        public static string TestProjectRoot { get; private set; }
```
Side idea: `PathTo` is not needed, we're in `Paths.` class and we'll invoke it like it everywhere.